meta-mender-qemu/Dockerfile

```diff
@@ -16,7 +16,7 @@
 # -e TENANT_TOKEN=<token>
 #       Use token as tenant token for client.
 
-FROM alpine:3.4
+FROM alpine:latest
```

Comment:
Hm noticed that qemu 2.9 is in edge already, we might not need this change after all.